packages/block-library/src/file/save.js

```diff
@@ -11,11 +12,35 @@ export default function save( { attributes } ) {
 		textLinkTarget,
 		showDownloadButton,
 		downloadButtonText,
+		displayPreview,
+		previewHeight,
 	} = attributes;
 
+	const embedLabel = RichText.isEmpty( fileName )
```

Comment:
It looks like it's only relevant to the PDF preview now. I got a bit confused seeing `PDF embed` hardcoded.

Reply:
I changed the variable name to make it a bit clearer. Since previews for other file types may be possible in the future, I [commented on the issue](https://github.com/WordPress/gutenberg/issues/19077#issuecomment-822973983) to the distinction between this PR, and the broader problem of file previews.

Reply:
Yes, it makes perfect sense, thank you 👍🏻